.github/workflows/release-code-freeze.yml

```diff
@@ -80,7 +80,7 @@ jobs:
   slack-notification:
     name: "Send notification to Slack"
     needs: [check-code-freeze, create-release-pr]
-    if: ${{ inputs.skipSlackPing != true }}
+    if: ${{ ! ( inputs.skipSlackPing && needs.create-release-pr.outputs.release-pr-created ) }}
```

Comment:
@jbordonado `continue-on-error` would work in this case, but there is a problem with the approach.

* If the release PR is created and build zip or smoke tests failed, we can make use of `continue-on-error` to send out the slack notification.
* But if the release PR creation failed for any reason (e.gbranch already exists), using `continue-on-error` would still send out the slack notification even if the PR is not created. Hence I added an output to confirm that the PR was ready before sending the slack alert.

WDYT?

Reply:
> But if the release PR creation failed for any reason (e.gbranch already exists), using continue-on-error would still send out the slack notification even if the PR is not created.

I thought we would add `continue-on-error: true` only on the `build-zip-and-run-smoke-tests` job inside `release-pr.yml`, meaning:
- the Slack job would run even when the `build-zip-and-run-smoke-tests` job inside `release-pr.yml` fails
- the Slack job would not run when the `prepare-release` job inside `release-pr.yml` fails, which is fine because we don't want to say on Slack that the release branch and PR are ready when it failed

But I realize now that this implies that if we trigger `release-pr.yml` manually and the smoke tests job fails, then we would not see this reflected on the workflow's run status, which may not be what we want (I'm not sure if that's important)

I'll let you decide if we keep it that way given these details

Reply:
I've added the workflow run link to the slack notification.